eventuals/concurrent.h

```diff
@@ -98,6 +95,9 @@ struct _Concurrent {
 
       // Each fiber forms a linked list of currently created fibers.
       std::unique_ptr<TypeErasedFiber> next;
+
+      // Need to store a cloned context in which would be stored callback.
+      std::optional<Context> child;
```

Comment:
Since we don't have a child/parent variable here let's do:
```suggestion
      std::optional<Context> context;
```